applications/ContactStructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -16,7 +16,7 @@
 // Project includes
 #include "includes/define.h"
 #include "includes/define_python.h"
-#include "custom_utilities/process_factory_utility.h"
+#include "custom_python/process_factory_utility.h"
```

Comment:
I think we should avoid those dependencies on python
What are you using it for?

Reply:
We have in the core the same dependencies for the python functions, it is similar

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/python_function_callback_utility.h

Reply:
The files are not new, I just move it for consistency, but when there is pybind dependency there no other way

Reply:
> https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/python_function_callback_utility.h

yep this one is the only "allowed" exception riccardo told me

Reply:
hm at second thought maybe you should leave it in utilities and manually add it
(like it is done in the core)
https://github.com/KratosMultiphysics/Kratos/blob/d78596fcfcf82354269119b3bb6d9f4c97954faa/kratos/CMakeLists.txt#L115-L117

Reply:
I prefer not, manual adding make less readable the CMakeList, and the point of this PR is making it simpler